package.json

```diff
@@ -1,76 +1,64 @@
-  },
-  "bugs": {
-    "url": "https://github.com/blackbaud/skyux-indicators/issues"
-  },
-  "homepage": "https://github.com/blackbaud/skyux-indicators#readme",
-  "peerDependencies": {
-    "@angular/common": "^9.1.6",
-    "@angular/core": "^9.1.6",
-    "@skyux/core": "^4.0.0",
-    "@skyux/i18n": "^4.0.0",
-    "@skyux/theme": "^4.15.2"
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "12.2.7",
+    "@angular/common": "12.2.7",
+    "@angular/compiler": "12.2.7",
+    "@angular/core": "12.2.7",
+    "@angular/forms": "12.2.7",
+    "@angular/platform-browser": "12.2.7",
+    "@angular/platform-browser-dynamic": "12.2.7",
+    "@angular/router": "12.2.7",
+    "@skyux/assets": "5.0.0-beta.2",
+    "@skyux/core": "5.0.0-beta.11",
+    "@skyux/e2e-client": "5.0.0",
+    "@skyux/i18n": "5.0.0",
+    "@skyux/theme": "5.0.0",
+    "rxjs": "6.6.7",
+    "tslib": "2.3.1",
+    "zone.js": "0.11.4"
   },
   "devDependencies": {
-    "@angular/animations": "9.1.13",
-    "@angular/common": "9.1.13",
-    "@angular/compiler": "9.1.13",
-    "@angular/compiler-cli": "9.1.13",
-    "@angular/core": "9.1.13",
-    "@angular/forms": "9.1.13",
-    "@angular/platform-browser": "9.1.13",
-    "@angular/platform-browser-dynamic": "9.1.13",
-    "@angular/router": "9.1.13",
-    "@blackbaud/auth-client": "2.48.0",
-    "@blackbaud/skyux-lib-clipboard": "4.0.0",
-    "@blackbaud/skyux-lib-code-block": "4.0.3",
-    "@blackbaud/skyux-lib-media": "4.0.0",
-    "@blackbaud/skyux-lib-restricted-view": "4.2.1",
-    "@blackbaud/skyux-lib-stache": "4.2.2",
-    "@skyux-sdk/builder": "4.9.0",
-    "@skyux-sdk/builder-plugin-skyux": "4.2.0",
-    "@skyux-sdk/e2e": "4.0.2",
-    "@skyux-sdk/testing": "4.3.0",
-    "@skyux/animations": "4.0.1",
-    "@skyux/assets": "4.0.1",
-    "@skyux/config": "4.4.0",
-    "@skyux/core": "4.6.2",
-    "@skyux/docs-tools": "4.9.1",
-    "@skyux/forms": "4.20.1",
-    "@skyux/http": "4.2.0",
-    "@skyux/i18n": "4.2.1",
-    "@skyux/indicators": "4.10.0",
-    "@skyux/inline-form": "4.1.1",
-    "@skyux/layout": "4.7.1",
-    "@skyux/lists": "4.8.0",
-    "@skyux/lookup": "4.17.0",
-    "@skyux/modals": "4.5.5",
-    "@skyux/omnibar-interop": "4.0.1",
-    "@skyux/popovers": "4.5.3",
-    "@skyux/router": "4.2.2",
-    "@skyux/tabs": "4.6.7",
-    "@skyux/theme": "4.19.0",
-    "codelyzer": "5.2.2",
-    "rxjs": "6.6.7",
+    "@angular-devkit/build-angular": "~12.2.7",
+    "@angular-eslint/builder": "12.3.1",
+    "@angular-eslint/eslint-plugin": "12.3.1",
+    "@angular-eslint/eslint-plugin-template": "12.3.1",
+    "@angular-eslint/schematics": "12.3.1",
+    "@angular-eslint/template-parser": "12.3.1",
+    "@angular/cli": "~12.2.7",
+    "@angular/compiler-cli": "~12.2.5",
+    "@skyux-sdk/documentation-schematics": "^5.0.0-beta.4",
+    "@skyux-sdk/e2e": "^5.0.0-beta.1",
+    "@skyux-sdk/testing": "^5.0.0-beta.2",
+    "@skyux/packages": "^5.0.0-beta.132",
```

Comment:
Range characters should be removed from all `@skyux/*`, `@skyux-sdk/*`, `@angular/*`, and `@angular-devkit/*`.

Also, `@skyux/packages` should be listed in `dependencies`.